src/main/java/no/sikt/nva/pubchannels/handler/fetch/FetchByIdentifierAndYearHandler.java

```diff
@@ -76,31 +72,23 @@ protected RequestInfo validate(RequestInfo requestInfo) {
         return requestInfo;
     }
 
-    protected ThirdPartyPublicationChannel fetchChannelFromChannelRegister(ChannelType type, String identifier, String year)
-        throws ApiGatewayException{
+    protected ThirdPartyPublicationChannel fetchChannelFromChannelRegister(ChannelType type, String identifier,
+                                                                           String year)
+        throws ApiGatewayException {
         try {
             LOGGER.info(FETCHING_FROM_CHANNEL_REGISTER_MESSAGE, type.name(), identifier);
             return publicationChannelClient.getChannel(type, identifier, year);
         } catch (PublicationChannelMovedException movedException) {
             throw new PublicationChannelMovedException(
                 "%s moved".formatted(type.name()),
-                constructNewLocation(getPathElement(type), movedException.getLocation(), year));
+                constructNewLocation(getPathElement(), movedException.getLocation(), year));
         }
     }
 
     protected boolean shouldUseCache() {
         return appConfig.shouldUseCache();
     }
 
-    private static String getPathElement(ChannelType type) {
```

Comment:
Did not see the use of this duplicate code, therefore implemented abstract method `protected abstract String getPathElement();`